lib/trento/activity_logging/activity_log.ex

```diff
@@ -7,6 +7,20 @@ defmodule Trento.ActivityLog.ActivityLog do
 
   @type t() :: %__MODULE__{}
 
+  @derive {
+    Flop.Schema,
+    filterable: [:type, :actor, :inserted_at],
+    sortable: [:type, :actor, :inserted_at],
+    max_limit: 100,
+    default_limit: 25,
+    default_order: %{
+      order_by: [:inserted_at],
+      order_directions: [:desc]
+    },
+    pagination_types: [:first, :last],
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do not leave false praise (which can actually be damaging). Do look for something to sincerely praise.  -->
**praise:** ‏I think first/last pagination is the only viable solution we have. Given the nature of the list, which is frequently populated (logs arrive at any time), limit/offset pagination would give some inconsistencies in the results.